tests/test_spectrometry.py

```diff
@@ -91,7 +91,21 @@ def test_gross_counts():
 def test_net_count():
     nc=sa.net_counts(gspec1, 475, 484, 1)
 
-
+def test_read_spec_id_file():
```

Comment:
How is this working? You don't seem to actually call `read_spec_id_file()`

Reply:
it's called at global scope